routes/index.js

```diff
@@ -136,4 +137,44 @@ router.get('/user/list/:page', function(req, res, next) {
     });
 });
 
+
+router.post('/user/address', function(req, res, next) {
+    var access_token = req.body.access_token;
```

Comment:
access_token is random no. its not equal user id? still ur using it like user id??